### ISSUE TITLE
Set Font size on attribute

### ISSUE THREAD

**Issue by vyscond** [astuetz/PagerSlidingTabStrip#187]
can we have it too?

**Comment by laylizzlle** [comment]
if its setting font size on the titles just define a style with the font size you want and set your pager to that style.It worked for me.

**Comment by vyscond** [comment]
Yeah, if you are crazy enough to buy this pattern from google. I really don't think it's "good" trade to write down a XML theme instead of write ```textSize="14sp"```

**Comment by laylizzlle** [comment]
Yeah I agree with you. I think he should add that attribute to the library
